drivers/mipi_dbi/Kconfig.stm32_fmc

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Bootlin
+# SPDX-License-Identifier: Apache-2.0
+
+config MIPI_DBI_STM32_FMC
+	bool "MIPI DBI driver for STM32 FMC"
+	default y
+	depends on DT_HAS_ST_STM32_FMC_MIPI_DBI_ENABLED
+	select MEMC
+	help
```

Nit- indent this line like so:

```suggestion
	help
	  Enable support for MIPI DBI driver for controller based on the stm32 FMC.
```